### ISSUE TITLE
Lines are truncated in docs

### ISSUE THREAD

**Issue by marilyndavis** [pandas-dev/pandas#17167]
[http://pandas.pydata.org/pandas-docs/stable/dsintro.html
]

Many lines in the ipython cells are truncated, which makes learning pandas extra frustrating and complicated.

![untitled](https://user-images.githubusercontent.com/15527675/28938863-f8bf267a-7854-11e7-93c5-b1be54b18510.png)

Expected is that docs would contain all the information needed for understanding and not be limited by the formatting.

**Comment by TomAugspurger** [comment]
Are you able to scroll over to the right?

**Comment by marilyndavis** [comment]
Yes, thank you!
Is there a way to change the formatting so that I can print it?  That's what I really like to do.
Again, thank you.

**Comment by TomAugspurger** [comment]
I'm not sure about printing from the HTML, that may depend on your browser and OS.

There is the PDF version at http://pandas.pydata.org/pandas-docs/stable/pandas.pdf that may print better (though it does have formatting errors of its own).

**Comment by marilyndavis** [comment]
Thank you again.  You mean all the \\\\\\\\\\\\\\\\\\\?  I guess extra stuff is a lot better than missing stuff.  It's only ugly.

Sometimes I think we are drowning in our tools.

So, should we consider this fixed?  I don't think so.  The pages should present correctly.  It's a huge waste of time, mind-energy, and frustration for anyone trying to learn Pandas.

Is that what you mean by "No action"?  It's not worth fixing?

**Comment by marilyndavis** [comment]
My OS: OS X 10.11.6  Both Safari and Chrome present badly.  Do things work well anywhere?  Should I dig up an old Windows box?

**Comment by TomAugspurger** [comment]
I think the issue is that your browser window is too narrow, not anything with pandas itself.

![screen shot 2017-08-03 at 3 28 14 pm](https://user-images.githubusercontent.com/1312546/28942288-6522e706-7860-11e7-8188-78f3aea5fc68.png)

**Comment by marilyndavis** [comment]
Wow thank you!  I cannot find a way to un-zoom text only in Safari, although instructions say I should.  But I did find a way to do it in Chrome.  And all is well now.
Thank you so so so much.

I guess, if a suggestion remains, it would be to tell people to look for a way to do this if the data are clipped.

Again.  Thank you so much Tom.

**Comment by jorisvandenbossche** [comment]
On my linux using both firefox or chrome, I get something like the figure @TomAugspurger posted above by default. So I think our docs itself are wide enough (we made it wider some time ago)

**Comment by marilyndavis** [comment]
Thank you.  Yes the docs are good.  But the possibility that someone might have their font so that the important bits are truncated is there.   I think it is worth mentioning this in a box where you have the first wide cell shown, but that's yours to decide.  I'm a teacher so I hate for silly details, unrelated to the subject, to get in the way of someone's learning.  
Thanks again for helping me.  I now have all the docs in one pdf that I made with pdftk.  I had to face python3 unicode pain to get that done.  But I'm all good now and ready to rock.
Marilyn
